src/main/java/internet/shop/service/impl/ProductServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package internet.shop.service.impl;
+
+import internet.shop.dao.ProductDao;
+import internet.shop.lib.Inject;
+import internet.shop.lib.Service;
+import internet.shop.model.Product;
+import internet.shop.service.ProductService;
+import java.util.List;
+
+@Service
+public class ProductServiceImpl implements ProductService {
+    @Inject
+    private ProductDao productDao;
+
+    @Override
+    public Product createProduct(Product product) {
```

Comment:
```suggestion
    public Product create(Product product) {
```
the fact that method accepts Object of type Product and that you are calling it from productServiceImpl already enough